### ISSUE TITLE
If indentation problem

### ISSUE THREAD

**Issue by deadtrickster** [elixir-lang/emacs-elixir#376]
Atom:

```elixir
if has_docs do
  docs_url = case latest_release? do
```

Emacs:

```elixir
if has_docs do
docs_url = case latest_release? do
```

Emacs version: GNU Emacs 25.0.94.1 (x86_64-pc-linux-gnu, GTK+ Version 3.18.9) of 2016-06-07
elixir-mode:       20160922.857

**Comment by deadtrickster** [comment]
ping :-)

**Comment by mattdeboard** [comment]
So `if` block indentation is broken across the board? Or what is different about this?

**Comment by whatyouhide** [comment]
@mattdeboard the content of the `do` block of the `if` is indented one level less than it should.

**Comment by mattdeboard** [comment]
Right, what I'm interested in is why this syntax specifically breaks indentation, but e.g.

```elixir
if has docs do
  x = 15
end
```

doesn't. In other words, trying to find the "edges" of this problem.

**Comment by whatyouhide** [comment]
Actually I just tried to indent this on master and this is what I get:

```elixir
if has_docs do
  docs_url = case latest_release? do
             end
end
```

which is correct as far as the `case` line goes. Then I am not a fan of the `end` placement and think it should be indented at the same level as `docs_url`, but that's another issue 😄

**Comment by dgutov** [comment]
@whatyouhide Indeed, the reported problem seems to be already fixed.
